README.md

```diff
@@ -39,6 +39,55 @@ library = foreman-ansible-modules/modules
 
 Now your playbooks and roles should have access to the modules contained in the repository for use, testing or development of new modules.
 
+## How to debug this repository
+
+Clone both ansible and this project
+
+```sh
+git clone https://github.com/ansible/ansible.git
+git clone https://github.com/theforeman/foreman-ansible-modules.git
+```
+
+Install python dependencies
+
+```sh
+cd ansible
+pip install -r requirements.txt
+cd ../foreman-ansible-modules
+pip install -r requirements.txt
+cd ..
+```
+
+Add a json input file for the module under test
+
+```sh
+$ cat activation-key-args.json
+{
+  "username":"admin",
+  "password":"changeme",
+  "server_url":"https://katello.example.com",
+  "name":"Katello Clients",
+  "organization":'"Default Organization"',
+  "lifecycle_environment":"Library",
+  "content_view":"client content view",
+  "subscriptions":[
+    {
+      "name":'"Red Hat Enterprise Linux"'
+    }
+  ]
+}
+```
+
+Debug with ansible's test-module
+
+```sh
+./ansible/hacking/test-module -m <path to ansible module> -a @<path to input arguments to module> -D <path to debugger>
+
+# Example: debug the activation_key module with pdb
+pip install pdb
+./ansible/hacking/test-module -m ./foreman-ansible-modules/modules/katello_activation_key.py -a @activation-key-args.json -D /usr/lib64/python2.7/pdb.py
```

Comment:
So, when I install it with `pip install pdb`, it's not automatically in my path. I believe the rpm does place it in my path, in which case you can do ``-D `which pdb` ``. I honestly used `mlocate`.